doc/intro.rst

```diff
@@ -40,6 +40,13 @@ will send notification to project subscribers if the test result metrics exceed
 a certain value. The threshold values will also appear in the charts. Projects 
 have the `project_settings` field for any specific configuration it might require.
 
+Builds can be compared against others, exposing regressions and fixes. Visit
+``/_/comparebuilds/``, select a project then two builds from it. This comparison
+will go over each build's tests and find all different states from each. For instance
+if a test failed for an older build but passes in a more current one, this is considered
```

Comment:
a test failed in baseline but passes in a current build...

Reply:
Done